video/out/vo.h

```diff
@@ -164,14 +164,12 @@ struct mp_pass_perf {
 };
 
 #define VO_PASS_PERF_MAX 64
+#define VO_PASS_DESC_MAX_LEN 128
```

Comment:
Seems quite low for some passes

Reply:
What would be sane size?

Reply:
We could dynamically allocate it and keep track of pointer or parent it to something, but then we could only free it at the "end". So we would essentially "leak" the memory on each `VOCTRL_PERFORMANCE_DATA` with move patch or every `info_callback` without that patch.